Makefile

```diff
@@ -346,6 +346,11 @@ $(BUILD_DIRECTORY)/$(APPLICATION_NAME_LOWERCASE)-$(APPLICATION_VERSION)-$(TARGET
 		-w $(BUILD_TEMPORARY_DIRECTORY) \
 		-o $@
 
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME_LOWERCASE)-$(APPLICATION_VERSION)-$(PLATFORM)-$(TARGET_ARCH_APPIMAGE).zip: \
+	$(BUILD_DIRECTORY)/$(APPLICATION_NAME_LOWERCASE)-$(APPLICATION_VERSION)-$(TARGET_ARCH_APPIMAGE).AppImage \
```

Comment:
Given that the "download page" at https://etcher.io talks about "x86" and "x64", do you think it might make sense to change this (and the corresponding targets) to:
```
$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(PLATFORM)-$(TARGET_ARCH).zip: \
	$(BUILD_DIRECTORY)/$(APPLICATION_NAME_LOWERCASE)-$(APPLICATION_VERSION)-$(TARGET_ARCH_APPIMAGE).AppImage \
```
so that e.g. `Etcher-1.1.3-linux-x86.zip` contains `etcher-1.1.3-i386.AppImage` ?
(doesn't really matter of course, and the existing convention means that https://github.com/resin-io/etcher-homepage wouldn't need changing)